app/models/rejection.rb

```diff
@@ -1,6 +1,7 @@
 class Rejection < ActiveRecord::Base
   REASONS = %w[
     slot_unavailable
+    no_adult
```

Comment:
I'm afraid we're going to have to add a column to https://github.com/ministryofjustice/prison-visits-2/blob/master/app/views/metrics/_all_prisons.html.erb for this...

Will need to check with @kyliemulholland